### PR TITLE
Back 2704 unit tests improvements

### PR DESCRIPTION
**Pull request by DennyGD**
Added tests coverage
Increased coverage for some modules
Added some constants
Tests refactoring

### CODE REVIEW

File: cmd/config.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const logger = require('../lib/logger.js');
 const project = require('../lib/project.js');
 const user = require('../lib/user.js');
 const util = require('../lib/util.js');
+const handleActionFailure = require('./../lib/util').handleActionFailure;
```
Comment (sjl2024) [inline cmd/config.js]: Nit: I would rename `handleActionFailure` to `handleCommandFailure` since it's more specific

**Comment by DennyGD** [comment]
Changed tests structure.
Current tests coverage stats:

**Comment by sjl2024** [comment]
@DennyGD LGTM. Looking it over one more time, it looks like some functions in util.js `readFile` are only aren't used anywhere else in the code and could probably be removed. Also, despite good coverage on util's APIs, it's a utility library so it would make sense to create standalone unit tests for it. But with the impending rewrite and integration tests, it might not be necessary - I'll leave it up to you

**Comment by DennyGD** [comment]
I hope to get a chance to refactor util.js during the refactor and add unit tests then.

**Pull request merged**